packages/quantic/force-app/main/default/lwc/quanticNoResults/quanticNoResults.js

```diff
@@ -8,21 +8,58 @@ import noResultsWithFilters from '@salesforce/label/c.quantic_NoResultsWithFilte
 import noResultsWithoutFilters from '@salesforce/label/c.quantic_NoResultsWithoutFilters';
 import undoLastAction from '@salesforce/label/c.quantic_UndoLastAction';
 
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").SearchStatus} SearchStatus */
+/** @typedef {import("coveo").HistoryManager} HistoryManager */
+/** @typedef {import("coveo").QuerySummary} QuerySummary*/
+/** @typedef {import("coveo").BreadcrumbManager} BreadcrumbManager */
+
+/**
+ * The `QuanticNoResults` component displays search tips and a "Cancel last action" button when there are no results. Any additional content slotted inside of its element will be displayed as well.
```

Comment:
I'm not sure about the `Any additional content slotted inside of...` part. What would you think of this?

    Any additional content embedded inside the component will be displayed as well.

Adding an example that shows how to embed additional content would be nice.

Reply:
This is taken from the atomic docs but I like your suggestion too. @jpmarceau what do you think?

Reply:
Yeah I prefer Luc's suggestion 👍

Reply:
Ok great! so we can make changes here that differ/improve on the descriptions taken from atomic and they'll be taken into account during the review you just mentioned?

Reply:
Yes exactly 😄